edit collaborations page (#6)
NCAR/geocat-hugo: content/collaborations/_index.md

---
title: "Collaborations"
date: 2022-04-27T00:00:00-06:00
image: images/banners/cloudSimulation.png

> description: "GeoCAT team members are involved in many other community software engineering efforts. Here are our most frequent collaborations:"

Reviewer: ```suggestion
description: "The GeoCAT team is involved in many community software engineering efforts. Here are our most frequent collaborations:"
```
